Content.Client/UserInterface/ParticleAccelerator/ParticleAcceleratorControlMenu.cs

```diff
@@ -460,14 +460,18 @@ private sealed class PASegmentControl : Control
             private readonly TextureRect _base;
             private readonly TextureRect _unlit;
             private readonly RSI _rsi;
+            private readonly int _pos;
+            private readonly string _afterUnderscore;
```

Comment:
i dont see why these need to be fields if they're only used in one function, just make them local variables

Reply:
Done